contributors/devel/sig-node/updating-pause-images.md

```diff
@@ -0,0 +1,30 @@
+# Updating Pause Image References
+
+The Kubernetes `pause` image that is used to run a 'parent' container for all
+the other containers inside your pod. This image is referenced across many
```

Comment:
Maybe reference these:

-  https://www.ianlewis.org/en/almighty-pause-container
- https://groups.google.com/forum/#!topic/kubernetes-users/jVjv0QK4b_o
- https://github.com/kubernetes-sigs/cri-o/issues/91

for more information